src/jsonschema.rs

```diff
@@ -71,6 +71,8 @@ struct Array {
 pub struct Tag {
     #[serde(rename = "type", default)]
     data_type: Value,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    format: Option<String>,
```

Comment:
It might be useful to treat format as a type instead of a String:

```rust
#[serde(...)]
type Format {
    DateTime
} 

pub struct Tag {
...
    format: Option<Format>
...
}


```